backend/app/views/spree/admin/orders/_adjustments.html.erb

```diff
@@ -1,21 +1,19 @@
-<% if adjustments.eligible.exists? %>
-  <fieldset class="no-border-bottom">
-    <legend><%= title %></legend>
-    <table>
-      <thead>
-        <tr>
-          <th><%= Spree::Adjustment.human_attribute_name(:name)%></th>
-          <th><%= Spree::Adjustment.human_attribute_name(:amount)%></th>
+<fieldset class="no-border-bottom">
+  <legend><%= title %></legend>
+  <table>
+    <thead>
+      <tr>
+        <th><%= Spree::Adjustment.human_attribute_name(:name)%></th>
+        <th><%= Spree::Adjustment.human_attribute_name(:amount)%></th>
```

Comment:
These should be translated:
Screenshot:
![Screenshot](https://www.dropbox.com/s/1klqnb48ir65bdg/Screenshot%202017-02-17%2009.21.09.png?dl=1)